src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -222,16 +240,22 @@ export default class Pilot {
      * @param {Number} speed - the speed to maintain, in knots
      * @return {Array} [success of operation, readback]
      */
-    maintainSpeed(currentSpeed, speed) {
+    maintainSpeed(currentSpeed, speed, aircraftModel) {
         const instruction = radio_trend('speed', currentSpeed, speed);
+        let speedUnattainable = '';
 
         this._mcp.setSpeedHold();
         this._mcp.setSpeedFieldValue(speed);
 
         // Build the readback
         const readback = {};
-        readback.log = `${instruction} ${speed}`;
-        readback.say = `${instruction} ${radio_spellOut(speed)}`;
+        readback.log = `${speedUnattainable}${instruction} ${speed}`;
+        readback.say = `${speedUnattainable}${instruction} ${radio_spellOut(speed)}`;
+        
+        if (speed > aircraftModel.model.speed.max || speed < aircraftModel.model.speed.min) {
```

Comment:
could we add this logic to the `AircraftTypeModel` class instead of doing this inline?  something like:
```javascript
// in AircraftTypeModel.js
isValidSpeed(nextSpeed) {
  return nextSpeed > this.model.speed.max || 
    nextSpeed < this.model.speed.min;
}
```

so then here we can simply use: 
```javascript
if (this.isValidSpeed(speed)) {
 // ...
}
```